translations/README.bosnian.md

```diff
@@ -0,0 +1,211 @@
+  - [Pogledaj API](https://realtime-poll.demo.hasura.app/console)
+
+### Video
+
+* [Dodajte GraphQL u samostojecu GitLab instancu](https://www.youtube.com/watch?v=a2AhxKqd82Q) (*3:44 min*)
+* [Todo aplikacija sa Auth0 i GraphQL backendom](https://www.youtube.com/watch?v=15ITBYnccgc) (*4:00 min*)
+* [GraphQL na GitLab-u integrirana sa GitLab autentikacijom](https://www.youtube.com/watch?v=m1ChRhRLq7o) (*4:05 min*)
+* [Nadzorna ploca za 10miliona voznji sa geografskom lokacijom (PostGIS, Timescale)](https://www.youtube.com/watch?v=tsY573yyGWA) (*3:06 min*)
+
+
+## Podrska & Rjesavanje problema
+
+Dokumentacija i zajednica ce vam pomoci da rijesite mnogo problema. Ako pronadjete gresku ili zelite da nas kontaktirate, mozete nas pronaci na kanalima ispod:
+
+* Podrska & povratne informacije: [Discord](https://discord.gg/vBPpJkS)
+* Problem & pracenje gresaka: [GitHub issues](https://github.com/hasura/graphql-engine/issues)
+* Pratite azuriranja o proizvodima: [@HasuraHQ](https://twitter.com/hasurahq)
+* Pricajte s nama na [nasoj web lokaciji](https://hasura.io)
+
+Zalazemo se za njegovanje otvorenog i dobrodoslog okruzenja u zajednici. Molimo da pogledate [kodeks ponasanja](code-of-conduct.md).
+
+Ako zelite prijaviti sigurnosni propust, molimo [procitajte ovo](SECURITY.md).
+
+## Doprinos
+
+Pogledajte nas [vodic za doprinos](CONTRIBUTING.md) za vise detalja.
+
+## Brendirani sadrzaj
+
+Hasura bendirani sadrzaj (logo, Hasura maskota, powered by bedzevi i slicno.) mogu biti pronadjeni u [assets/brand](assets/brand) mapi. Slobodno ih mozete koristiti u vasoj aplikaciji/web stranici. Biti ce nam drago i ako dodate "Powered by Hasura" bedz na vasu aplikaciju koju ste napravili uz pomoc Hasura-e. ❤️
+
+<div style="display: flex;">
+  <img src="assets/brand/powered_by_hasura_blue.svg" width="150px"/>
+  <img src="assets/brand/powered_by_hasura_white.svg" width="150px"/>
+</div>
+
+```html
+<!-- Za svijetle pozadine -->
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_blue.svg" />
+</a>
+
+<!-- Za tamne pozadine -->
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_white.svg" />
+</a>
+```
+
+## Licenca
+
+GraphQL Engine je dostupan pod [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) (Apache-2.0).
+
+Sav **ostali sadrzaj** (osim onog u [`server`](server), [`cli`](cli) i
+[`console`](console) mapama) su dostupni pod [MIT License](LICENSE-community).
+To ukljucuje sve u [`docs`](docs) i [`community`](community)
+mapama.
+
+## Prijevodi
+
+Ova stranica je dostupna u sljedecim jezicima:
+
+- [Japanski :jp:](translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
+- [Francuski :fr:](translations/README.french.md) (:pray: [@l0ck3](https://github.com/l0ck3))
+- [Bosanski :bs:](translations/README.bosnian.md) (:pray: [@hajro92](https://github.com/hajro92))
```

Comment:
We don't need to list the Bosnian translation here, as we are already in this file. But please add the English translation to the list here 🙂